jmx-metrics/src/main/resources/target-systems/hbase.groovy

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+def beanMasterServer = otel.mbeans("Hadoop:service=HBase,name=Master,sub=Server")
+otel.instrument(beanMasterServer, "hbase.master.region_server.count",
+  "The number of region servers.", "{servers}",
+  ["numDeadRegionServers":["state" : {"dead"}], "numRegionServers": ["state" : {"live"}]],
+  otel.&longUpDownCounterCallback)
+
+def beanMasterAssignmentManager = otel.mbean("Hadoop:service=HBase,name=Master,sub=AssignmentManager")
+otel.instrument(beanMasterAssignmentManager, "hbase.master.in_transition_regions.count",
```

Comment:
nit, `regions_in_transition` is more readable imo

Reply:
Updated